vulnerabilities/importers/redhat.py

```diff
@@ -65,31 +67,74 @@ def fetch():
 
 
 def to_advisory(advisory_data):
-
     affected_purls = []
     if advisory_data.get("affected_packages"):
         for rpm in advisory_data["affected_packages"]:
             if rpm_to_purl(rpm):
                 affected_purls.append(rpm_to_purl(rpm))
 
     references = []
-    if advisory_data.get("bugzilla"):
-        bugzilla = advisory_data.get("bugzilla")
+    bugzilla = advisory_data.get("bugzilla")
+    if bugzilla:
+        url = "https://bugzilla.redhat.com/show_bug.cgi?id={}".format(bugzilla)
+        bugzilla_data = requests.get(f"https://bugzilla.redhat.com/rest/bug/{bugzilla}").json()
```

Comment:
Just as a side note, this is the kind of JSON we would likely need to store forever as back auditable evidence when we will do this later ... which likely calls for a central place where we fetch things from